pgbouncer-mixin/panels.libsonnet

```diff
@@ -0,0 +1,156 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+{
+  new(this):
+    {
+      local t = this.grafana.targets,
+      local stat = g.panel.stat,
+      // create stat panel using commonlib
+      clientsWaitingConnections:
+        commonlib.panels.generic.stat.base.new(
+          'Client waiting connections',
+          targets=[t.clientsWaitingConnections],
+          description='Current number of client connections waiting on a server connection.'
+        )
+        + stat.options.withGraphMode('none')
+        + stat.standardOptions.color.withMode('thresholds')
+        + stat.standardOptions.thresholds.withSteps([
+          stat.thresholdStep.withColor('super-light-green')
+          + stat.thresholdStep.withValue(0),
+          stat.thresholdStep.withColor('super-light-orange')
+          + stat.thresholdStep.withValue(10),
+          stat.thresholdStep.withColor('super-light-red')
+          + stat.thresholdStep.withValue(20),
+        ]),
+      activeClientConnections:
+        commonlib.panels.generic.stat.info.new(
+          'Active client connections',
+          targets=[t.activeClientConnections],
+          description='Current number of active client connections.'
+        )
+        + stat.options.withGraphMode('none'),
+      activeServerConnections:
+        commonlib.panels.generic.stat.info.new(
+          'Active server connections',
+          targets=[t.activeServerConnections],
+          description='Current number of client connections that are linked to server connection and able to process queries.'
```

Comment:
"to server connections" or "to a server connection"? Also just verifying that this actually. makes sense. How does a client connection "link" to a server connection? They're not the same connection?

Reply:
gotcha